### DISCUSSION TITLE
Docker Installation and setup

### DISCUSSION THREAD

**Discussion by pcaveness** [hnimminh/libresbc#156] [General]
It seems a bit confusing when installingfrom docker that it doesn't pull everything necessary to run LibreSBC.

**You must first clone the git and then perform the docker pull.**
This is different from normal, where pulling an image from docker it will collect everything you need to run the container.
You have to run the line provided in the docker page from the (git pull)/build/docker/ directory.

After doing these steps, the docker page shows that you access the UI by running 
`./libreui-linux-amd64 -L http://127.0.0.1:8080 -H 0.0.0.0 -P 9099`
but i cannot find ./libreui-linux-amd64 anywhere in the git and it does not seem to be mentioned anywhere.
I can run the webui like normal from the /webui/ directory but that doesnt match.

Also the slack channel is invite only. Is there a discord or way to request an invite?

**Comment by hnimminh** [comment]
@pcaveness `You must first clone the git and then perform the docker pull.` nope, seem you're doing something wrong. 
simply run `docker pull hnimminh/libresbc` that's all

The document for UI is not clear, PR is always welcome.

**Comment by pcaveness** [comment]
okay, i was unaware that i needed to create libre.env file separate. i see that now.
where is libreui-linux-amd64 ?

**Reply by hnimminh** [reply]
With a help from nice guys, https://github.com/hnimminh/libresbc/discussions/120
